packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -113,10 +116,16 @@ contract MixinExchangeCore is
             require(order.takerAssetAmount > 0);
             require(isValidSignature(orderHash, order.makerAddress, signature));
         }
+        
+        // Validate sender
+        if (order.senderAddress != address(0)) {
```

Comment:
Let's set this conditional to a helpful variable name. Perhaps:

```
const isDirectTakerFill = order.senderAddress != address(0);
```

It's helpful to have a human-readable variable name for conditionals that elucidate the intent behind the conditional.

If we don't want to create intermediate variables because of stack limit, let's add a more helpful comment.

Reply:
Yeah, we're pretty much right at the stack limit unfortunately, but I can add a more descriptive comment.